src/Routes.tsx

```diff
@@ -14,12 +14,41 @@ import { CommonRoutesFeatures } from './RoutesWrap';
 const { MainView } = lazily(() => import('~views/Main/Main'));
 const { Reports } = lazily(() => import('~views/Reports/Reports'));
 const { ReportPage } = lazily(() => import('~views/Report/Report'));
+const { AboutPage } = lazily(() => import('~views/About/About'));
 const { BivariateManagerPage } = lazily(
   () => import('~views/BivariateManager/BivariateManager'),
 );
 
 export function RoutedApp() {
   const [{ data: userModel, loading }] = useAtom(userResourceAtom);
+
+  function isFirstTimeVisit() {
```

Comment:
function name is like it's just checker, but it mutates, I would expect this function to be clean, just return boolean, and sideeffect logic in useEffect